torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2857,19 +2857,15 @@ Tensor linalg_qr_backward(const std::vector<torch::autograd::Variable> &grads, c
       rhs_term = at::matmul(Q, M);
     }
 
-    // We want to compute: (rhs_term @ R^{-H})
-    // Note that (rhs_term @ R^{-H}) = (R^{-1} @ rhs_solve_1^H)^H
-    // Since R is upper triangular, we can do this using
-    // triangular_solve(rhs_term^H, R)^H
-    Tensor grad_A;
-    std::tie(grad_A, std::ignore) = at::triangular_solve(
-        rhs_term.mH(),
-        R,
-        /*upper=*/true,
-        /*transpose=*/false,
+    // Compute rhs_term @ R^{-H}
+    Tensor grad_A = at::linalg_solve_triangular(
+        R.transpose(-2, -1).conj(),
```

Comment:
can we just call `R.adjoint()` instead?

also looks like `adjoint`'s documentation doesn't render properly https://pytorch.org/docs/master/generated/torch.Tensor.adjoint.html#torch.Tensor.adjoint

Reply:
It renders the same as all the other tensor methods (for good or bad...)

And sure, I'll change it into using `mH()` once I go over this function again when moving `lu_solve` to `linalg.`.

Reply:
ok I meant I don't see docs for `torch.adjoint`. Probably missing an entry in some rst file.

Reply:
Would you file an issue?

Reply:
oh, I had not realised that clicking on adjoint does not take you anywhere. I'll write a fix.

Reply:
https://github.com/pytorch/pytorch/pull/68869